internal/api/web/handlers/quiz.go

```diff
@@ -12,13 +15,42 @@ import (
 func RegisterQuizHandlers(e *echo.Echo) {
 	e.GET("/", quizHomePage)
 	e.GET("/quizpage", GetQuizPage)
+	e.GET("/nextquestion", GetIsCorrect)
+	e.POST("/nextquestion", POSTNextQuestion)
 }
 
 // Renders the quiz home page
 func quizHomePage(c echo.Context) error {
 	return utils.Render(c, http.StatusOK, quiz_pages.QuizHomePage())
 }
 
+var questionIndex = 0
+
 func GetQuizPage(c echo.Context) error {
-	return utils.Render(c, http.StatusOK, quiz_pages.QuizPage())
+	sampleQuiz := quizzes.SampleQuiz.Questions[questionIndex]
+	title := quizzes.SampleQuiz.Title
+
+	return utils.Render(c, http.StatusOK, quiz_pages.QuizPage(sampleQuiz, title))
+}
+
+func GetIsCorrect(c echo.Context) error {
+	answer := c.QueryParam("answer")
+	correct := ""
+	alternatives := quizzes.SampleQuiz.Questions[questionIndex].Alternatives
+	for _, aswr := range alternatives {
+		if aswr.IsCorrect {
+			correct = aswr.Text
+		}
+	}
+
+	fmt.Println(answer)
+	return utils.Render(c, http.StatusOK, quiz_components.Answers(alternatives, quiz_components.CorrectAndAnswered(correct, answer)))
```

Comment:
Implent getting this info from the DB, That should be up and running on the latest commit on dev.